### ISSUE TITLE
Give a brief overview of how that actually works

### ISSUE THREAD

**Issue by okomarov** [emmethalm/infiniteGPT#4]
It's unclear how you achieve that and looking at other issues it seems there might be a discrepancy on expectations. 

A brief overview of what the script actually does would help narrow down what use cases it serves

**Comment by ghauw** [comment]
Yes. I've also written a modification that prompts the user to paste what they want to ask into the prompt which then writes it to a file.

**Comment by zenflow** [comment]
@okomarov read the source code.. it will take < 10 seconds .. not sure but I think this repo is a joke or something

**Comment by okomarov** [comment]
@zenflow I was giving the author the benefit of the doubt

**Comment by mpottinger** [comment]
> @okomarov read the source code.. it will take < 10 seconds .. not sure but I think this repo is a joke or something

Probably a joke. I looked at the source code and was like "Ok. What is this going to do except waste tokens?!?

Then I realized this is for doing bulk operations on the text. It is for cleaning grammar, spelling, punctuation, translation, or whatever over the entire text. Whatever you want. If you want to convert the whole thing to caveman speak you could do that.
